src/app/project/[id]/page.tsx

```diff
@@ -44,17 +44,17 @@ export default function Project({ params }: { params: { id: string } }) {
       </CardSection>
       <CardSection displaySectionTitle sectionTitle={{ title: t(`sectionHeaders.verificationValidation`) }}>
         <Container flex={2} variant="cardSectionNoMargin">
-          <>Lorem ispum</>
+          <>{t(`lorem`)}</>
         </Container>
       </CardSection>
       <CardSection displaySectionTitle sectionTitle={{ title: t(`sectionHeaders.issuancesRetirements`) }}>
         <Container flex={2} variant="cardSectionNoMargin">
-          <>Lorem ispum</>
+          <>{t(`lorem`)}</>
         </Container>
       </CardSection>
       <CardSection displaySectionTitle sectionTitle={{ title: t(`sectionHeaders.documents`) }}>
         <Container flex={2} variant="cardSectionNoMargin">
-          <>Lorem ispum</>
+          <>{t(`lorem`)}</>
```

Comment:
Are you sure for the "lorem"? If that's for when page is loading, I would say it's better to rely on skeleton

Reply:
This is to display text in the empty div awaiting user story refinement.